code/__DEFINES/_macros.dm

```diff
@@ -127,11 +127,13 @@
 #define from_target(target, receiver)                       target >> (receiver)
 #define to_file(file_entry, file_content)                   file_entry << file_content
 
+#define place_meta_charset(content) istext(content) ? "<meta charset=\"utf-8\">" + content : content
```

Comment:
```suggestion
#define (place_meta_charset(content) istext(content) ? "<meta charset=\"utf-8\">" + (content) : (content))
```

Reply:
The marco parameters and the macro itself should be parenthesized to avoid issues.
See here for an example: https://wiki.sei.cmu.edu/confluence/display/c/PRE01-C.+Use+parentheses+within+macros+around+parameter+names
(I know its C, but the general principle also applies to the byond preprocessor)